src/commands/lint.ts

```diff
@@ -73,11 +72,11 @@ export class LintCommand implements Command {
     }
   ];
 
-  run(options, config): Promise<any> {
+  run(options: CommandOptions, config: ProjectConfig): Promise<any> {
     // Defer dependency loading until this specific command is run
     const polylint = require('polylint/lib/cli');
 
-    let lintFiles: string[] = options.input;
+    let lintFiles: string[] = options['input'];
```

Comment:
The object is used as a dictionary, so this syntax represents its use the best. We could create a type per command, but I wanted to do a small change to enable `noImplicitAny`.